packages/mds-trip-processor/src/proc-trip.ts

```diff
@@ -89,16 +89,14 @@ async function processTrip(
     }
 
     await db.insertTrips(tripData)
-    // Delete all processed telemetry data and update cache
-    delete telemetryMap[trip_id]
-    await cache.writeTripsTelemetry(`${provider_id}:${device_id}`, telemetryMap)
-
-    return trip_id
+    await cache.deleteTripEvents(`${provider_id}:${device_id}:${trip_id}`)
```

Comment:
I dig the cache cleanup here, just wanna call out that this could potentially have weird interactions if we have ongoing writes from the `event-processor` at `trip-processor` runtime.  Not sure the best way to resolve that.

Reply:
Agreed, I want to look into how to handle Redis transactions best, specifically the WATCH method.